src/bundle/Resources/views/themes/admin/content/edit_base.html.twig

```diff
@@ -62,7 +62,7 @@
                     </div>
                 {% else %}
                     <div class="row">
-                        <div class="offset-1 col-6 p-0">
+                        <div class="{{ is_full_width|default(false) ? 'col-12' : 'col-6 offset-1 p-0' }}">
```

Comment:
I know it's not yours but could we not use the class `p-0` and the like because later there are more problems than good